gene2phenotype_project/gene2phenotype_app/views/gencc_submission.py

```diff
@@ -0,0 +1,49 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import generics, permissions
+from rest_framework import status
+from drf_spectacular.utils import extend_schema
+
+from gene2phenotype_app.serializers import (
+    GenCCSubmissionSerializer,
+    G2PStableIDSerializer,
+    CreateGenCCSubmissionSerializer,
+)
+
+@extend_schema(exclude=True)
+class GenCCSubmissionCreateView(generics.CreateAPIView):
+    serializer_class = CreateGenCCSubmissionSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+@extend_schema(exclude=True)
+class GenCCSubmissionView(APIView):
+    """GenCCSubmissionView"""
+
+    def get(self, request) -> Response:
+        """Gets the unsubmitted stable ids
+
+        Returns:
+            Response: Response containing the status and the serializer.data
+        """
+        unused_ids = GenCCSubmissionSerializer.fetch_list_of_unsubmitted_stable_id()
+        serializer = G2PStableIDSerializer(unused_ids, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
```

Comment:
The output format could be simpler:
```
[
    {
        "stable_id": "G2P00016"
    },
    {
        "stable_id": "G2P00033"
    }
]
```
In the end this should be a list of stable ids, there is no need to have a dictionary.

Also, to be consistent the output format should be the same as `StableIDsWithLaterReviewDateView`.